### PR TITLE
Add localization messages for 'pt-BR' and 'es' locales

### PR DESCRIPTION
**Pull request by pedrofurtado**
@BingoRUS  This PR came from this another PR: https://github.com/DevExpress/bower-devextreme/pull/15

### CODE REVIEW

**Review by BingoRUS — CHANGES_REQUESTED** [review]
@pedrofurtado I've added 2 commits with the new localization constants and empty messages for 17.2 features. Could you please take a look and add translations there?

**Comment by pedrofurtado** [comment]
@BingoRUS I've made the requested changes. Tell me if something must be modified or added in order to this PR be able to be merged.

File: js/localization/messages/es.json [hunk 1]
```diff
@@ -195,38 +195,38 @@
         "dxScheduler-confirmRecurrenceEditOccurrence": "Editar cita",
         "dxScheduler-confirmRecurrenceDeleteOccurrence": "Eliminar cita",
         "dxScheduler-noTimezoneTitle": "Sin zona horaria",
-        "dxScheduler-moreAppointments": "",
+        "dxScheduler-moreAppointments": "{0} +",
```
Comment (BingoRUS) [inline js/localization/messages/es.json]: Could we use "{0} más" here?

File: js/localization/messages/pt-BR.json [hunk 2]
```diff
@@ -195,38 +195,38 @@
         "dxScheduler-confirmRecurrenceEditOccurrence": "Editar compromisso",
         "dxScheduler-confirmRecurrenceDeleteOccurrence": "Eliminar compromisso",
         "dxScheduler-noTimezoneTitle": "Sem fuso horário",
-        "dxScheduler-moreAppointments": "",
+        "dxScheduler-moreAppointments": "{0} +",
```
Comment (BingoRUS) [inline js/localization/messages/pt-BR.json]: Could we use "{0} mais" here?

**Comment by BingoRUS** [comment]
@pedrofurtado Thank you! Just a quick question about "dxScheduler-moreAppointments" constant. We use "{0} more" for English localization. Is it possible to use something similar from Spanish and Portugal? I added comments with examples above.

File: js/localization/messages/es.json (hunk 1 above)
Comment (pedrofurtado) [reply js/localization/messages/es.json]: @BingoRUS Yes, that is another option for this translation. Let's use your suggestion, but both are valid. I will change it here.

File: js/localization/messages/pt-BR.json (hunk 2 above)
Comment (pedrofurtado) [reply js/localization/messages/pt-BR.json]: @BingoRUS Yes, that is another option for this translation. Let's use your suggestion, but both are valid. I will change it here.

**Comment by pedrofurtado** [comment]
@BingoRUS Requested changes are done.

**Review by BingoRUS — APPROVED** [review]

**Comment by BingoRUS** [comment]
@pedrofurtado Your pull request is merged now. Thanks!

**Pull request merged**